docs/fr/docs/advanced/response-directly.md

```diff
@@ -1,63 +1,63 @@
-# Return a Response Directly
+# Retourner une réponse directement
 
-When you create a **FastAPI** *path operation* you can normally return any data from it: a `dict`, a `list`, a Pydantic model, a database model, etc.
+Lorsque vous créez une *opération de chemins* **FastAPI**, vous pouvez normalement retourner n'importe quelle donnée : un `dict`, une `list`, un modèle Pydantique, un modèle de base de données, etc.
 
-By default, **FastAPI** would automatically convert that return value to JSON using the `jsonable_encoder` explained in [JSON Compatible Encoder](../tutorial/encoder.md){.internal-link target=_blank}.
+Par défaut, **FastAPI** convertirait automatiquement cette valeur de retour en JSON en utilisant le `jsonable_encoder` expliqué dans [JSON Compatible Encoder](../tutorial/encoder.md){.internal-link target=_blank}.
 
-Then, behind the scenes, it would put that JSON-compatible data (e.g. a `dict`) inside of a `JSONResponse` that would be used to send the response to the client.
+Ensuite, dans les coulisses, il mettra ces données compatibles JSON (par exemple un `dict`) à l'intérieur d'un `JSONResponse` qui sera utilisé pour envoyer la réponse au client.
```

Comment:
I will leave the word `JSON-compatible` in place of `compatibles-JSON`.

Reply:
I would have translated `behind the scenes` by the word `en arrière-plan` or `en coulisses`.

Reply:
I correct this, if it's ok for you, please, close the comment. Thanks.